lib/std/http/Client.zig

```diff
@@ -10,6 +10,7 @@ const testing = std.testing;
 const http = std.http;
 const mem = std.mem;
 const net = std.net;
+const socks5 = std.socks.@"5";
```

Comment:
I haven't see files named after numbers only, maybe `v5` is better?

Reply:
This new name is good. Originally I consider "5" might be shorter, but no luck.